packages/vm/src/context.rs

```diff
@@ -39,7 +39,7 @@ impl GasState {
     }
 
     #[allow(unused)]
-    fn use_gas(&mut self, amount: u64) {
+    fn increase_externally_used_gas(&mut self, amount: u64) {
```

Comment:
clearer name. I had to read the code here to understand what it was doing
(it made sense when I reviewed the original PR 3 times as all context was in my head, but coming back a month later, nicer names help)